### ISSUE TITLE
Can't create S3 blob store with Docker Nexus3

### ISSUE THREAD

**Issue by lazycodeninja** [sonatype/nexus-blobstore-s3#13]
Thanks for creating an issue! Please fill out this form so we can be
sure to have all the information we need, and to minimize back and forth.

* **What are you trying to do?**

I've extended the official Nexus3 Docker container with this plugin, and I'm unable to create an S3 bucket blob store.  Originally I was using a snapshot of 1.0.1, and I was getting this error:

```
The bucket is in this region: ap-northeast-1.Please use this region to retry the request (Service: Amazon S3; Status Code: 301; Error Code: PermanentRedirect...
```
I couldn't work out where `ap-northeast-1` came from, as my AWS config region is `ap-southeast-2`, which is where my S3 bucket is.

I then tried rebuilding the Docker container after a `git pull` and now get a different error:

```
2017-09-18 05:05:32,218+0000 ERROR [qtp1245966767-66] admin org.sonatype.nexus.extdirect.internal.ExtDirectServlet - Failed to invoke action method: coreui_Blobstore.create, java-method: org.sonatype.nexus.coreui.BlobStoreComponent.create
java.lang.IllegalStateException: Missing: {s3} bucket
	at com.google.common.base.Preconditions.checkState(Preconditions.java:444)
	at org.sonatype.nexus.common.collect.AttributesMap.require(AttributesMap.java:198)
	at org.sonatype.nexus.blobstore.s3.internal.S3BlobStore.getConfiguredBucket(S3BlobStore.java:432)
	at org.sonatype.nexus.blobstore.s3.internal.S3BlobStore.init(S3BlobStore.java:408)
	at org.sonatype.nexus.repository.internal.blobstore.BlobStoreManagerImpl.newBlobStore(BlobStoreManagerImpl.java:201)
...
```
I've confirmed that the credentials and permissions are good by running a `/bin/bash` shell on the Docker container and running some AWS CLI commands on the bucket.  They all work fine.

* **What feature or behavior is this required for?**
Base functionality, creating an S3 blob store.

* **How could we solve this issue? (Not knowing is okay!)**

* **Anything else?**

Here's my Dockerfile:
```
FROM sonatype/nexus3:3.5.2

ENV S3_BLOBSTORE_VERSION 1.1.0-SNAPSHOT
ENV NEXUS_HOME /opt/sonatype/nexus

COPY target/nexus-blobstore-s3-${S3_BLOBSTORE_VERSION}.jar ${NEXUS_HOME}/system/org/sonatype/nexus/nexus-blobstore-s3/${S3_BLOBSTORE_VERSION}/

USER root

RUN sed -i.bak \
  -e "/nexus-blobstore-file/a\\"$'\n'"<bundle>mvn:org.sonatype.nexus/nexus-blobstore-s3/${S3_BLOBSTORE_VERSION}</bundle>" \
  ${NEXUS_HOME}/system/org/sonatype/nexus/assemblies/nexus-base-feature/*/nexus-base-feature-*-features.xml \
  ${NEXUS_HOME}/system/org/sonatype/nexus/assemblies/nexus-core-feature/*/nexus-core-feature-*-features.xml

RUN yum -y update && yum install -y python-setuptools && easy_install pip && pip install awscli

USER nexus
```
After building the image, I run the container with the command
```
docker run -d -p8081:8081 --name nexus -v nexus-data:/nexus-data -v ~/Dev/nexus-blobstore-s3/aws:/nexus-data/.aws aconex/nexus3-s3:3.5.2
```

**Comment by vedit** [comment]
I also am having problems with the same exact exceptions but I have manually installed the nexus.
Nexus Version: 3.5.2
S3 Blobstore Version: 1.10-SNAPSHOT

When I try to create an s3 blobstore it fails like the image below.
![image](https://user-images.githubusercontent.com/691858/30541478-38292560-9c84-11e7-9d60-36611c36bea0.png)

Thrown exception is the exact same with the OP.

What nexus version is compatible with blobstore-s3?

**Comment by ataylor284** [comment]
Sorry, this got broken when some nexus 3.6 stuff accidentally got merged.  Fixed by https://github.com/sonatype/nexus-blobstore-s3/commit/77d7c7b500a8df63fe03c57a1ddb1d59e91520cb.  Also note that Nexus 3.6 has been released and the nexus-3.6 branch has been merged (also fixing this issue).
